src/test/run-pass-fulldeps/auxiliary/proc_macro_def.rs

```diff
@@ -8,47 +8,37 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-#![feature(plugin, plugin_registrar, rustc_private)]
```

Comment:
Existing procedural macros will lose the `syntax::tokenstream::TokenStream` quoter from `proc_macro_internals`, AFAIK other than that they won't have to make any changes. I could include a `syntax::tokenstream::TokenStream` quoter for back-compat.

How many people are actually using "existing procedural macros" though? This PR doesn't effect the widely used legacy plugin system; I didn't think many people were using `SyntaxExtension::ProcMacro` from #36154.